### ISSUE TITLE
CopyQ not showing up (Windows 7)

### ISSUE THREAD

**Issue by 4344967741** [hluk/CopyQ#420]
I've been trying to setup copyq on windows for the past two days, I can instal it flawlessly but then nothing
If I launch the application, it goes in the tray, but doesn't record the clipboad (Hovering the mouse above the icon just shows EMPTY) and clicking on said icon doesn't do anything, no interface, nothing.

Does anyone know what could be wrong? I'm really eager to try this, thanks in advance

**Comment by scottkosty** [comment]
When you run copyq from the terminal, are there informative messages? You can get even more information if you build with `-DCMAKE_BUILD_TYPE=Debug`.

**Comment by hluk** [comment]
You should be able to get lot more info by running following commands in "powershell" (no recompilation required).

```powershell
cd ~
$env:COPYQ_LOG_LEVEL='TRAC'
$env:COPYQ_LOG_FILE='copyq.log'
& 'C:\Program Files (X86)\CopyQ\copyq.exe'
```

This will create log file in your home directory (possibly in "C:\User\<user-name>\copyq.log").

**Comment by hluk** [comment]
Should be fixed in v2.6.1 (at least I fixed very similar issue). Reopen if not fixed.
